pkgs/servers/owncast/default.nix

```diff
@@ -1,43 +1,31 @@
-{ lib, buildGoModule, fetchFromGitHub, nixosTests, bash, which, ffmpeg, makeWrapper, coreutils, ... }:
-
-buildGoModule rec {
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+, ffmpeg
+, makeBinaryWrapper
+}:
+
+let
+  version = "0.1.1";
```

Comment:
`version` is only used once, there's no need to have a `let/in` then.

Reply:
Isn't it used twice: inherited in `buildgomodule` and also used to build the `rev` in `fetchFromGitHub`?

Reply:
@wlcx take a look at how it was before this PR :)
`rev = "v${version}";` works without a let in.

Reply:
You're right, sorry. Ignore my previous comment.

Reply:
cc @MayNiklas who also commented on this.

Unless I'm missing something, I thought the two options here were to use `rec` around the `buildGoModule` function or use a `let... in` like I have done, and I thought the general consensus was that the latter was better?

Reply:
argh, sorry for out of order comments 

@MayNiklas yes, but that used `rec` :-)